dts/bindings/pwm/st,stm32-pwm.yaml

```diff
@@ -14,9 +14,10 @@ properties:
       description: Clock prescaler at the input of the timer
 
     "#pwm-cells":
-      const: 2
+      const: 3
 
 pwm-cells:
   - channel
 # period in terms of nanoseconds
   - period
+  - flags
```

Comment:
flags is either `PWM_POLARITY_NORMAL` or  `PWM_POLARITY_INVERTED` , correct ?

Reply:
Yes, they're defined `#include <dt-bindings/pwm/pwm.h>`. I guess that flags could potentially be used for other purposes in the future.